web/templates/web/study-detail.html

```diff
@@ -36,6 +37,11 @@
         function childSelected(selected) {
             var participateButton = document.getElementById('participate-button');
             selected.value === 'none' ? participateButton.disabled = true : participateButton.disabled = false;
+            participateButton.value = selected.value;
+        }
+
+        function portToParticipate(selected) {
```

Comment:
i changed the preview route so it opens in a new tab. this was for the participate route - not sure if we should change that one or not. the player is supposed to have things to allow you to exit early and then it directs you back to the exitUrl that's defined in the study.